### DISCUSSION TITLE
A setting so when you close Alienfx it minimizes instead

### DISCUSSION THREAD

**Discussion by sw0277** [T-Troll/alienfx-tools#326] [Ideas]
It would be nice to see a setting that when enabled, if you closed Aleinfx it would minimize itself instead. I have Alienfx to automatically start with Windows so that its always running. But, when I'm done adjusting things, I sometimes accidentally close it instead of minimizing it and don't realize it until later.

**Comment by T-Troll** [comment]
Since you're not the first who telling this, i'll do something about it into next release.

**Reply by sw0277** [reply]
Perfect, the newest update 8.2.6.1 works great. Now I don't have to worry about accidentally closing Alienfx. Thanks!
